.github/workflows/pr_bot.yml

```diff
@@ -18,7 +18,7 @@ jobs:
             var msg = `Thanks for opening a PR! :100: Please note that [all commits must be signed off](https://magma.github.io/magma/docs/next/contributing/contribute_workflow#guidelines). This is enforced by the \`DCO check\`.
 
             ### Howto
-
+            - ***PR Title.*** Follow [conventional commits format](https://docs.magmacore.org/docs/next/contributing/contribute_workflow#guidelines). This is enforced by \`Semantic Pull Request\` check.
```

Comment:
do you mind adding the word Semantic so if semantic check fails they can associate this?

***PR Title.*** Follow semantics at [conventional commits format]